string primitive_name
KeyValue[] params
---
bool success